tests/unit/drivers/test_concat_driver.py

```diff
@@ -30,12 +30,12 @@ class ConcatDriverTestCase(JinaTestCase):
 
     def test_direct_concat(self):
         doc1, doc2 = input_fn()
-
         t1 = np.concatenate([pb2array(doc1.embedding), pb2array(doc2.embedding)], axis=0)
         doc1.embedding.buffer += doc2.embedding.buffer
         doc1.embedding.shape[0] += doc2.embedding.shape[0]
         t2 = pb2array(doc1.embedding)
-
+        self.assertEqual(t1.shape[0], 10)
+        self.assertEqual(t2.shape[0], 10)
```

Comment:
can we slowly replace these unittest style assert into pytest style?
into:

```python
assert t1.shape[0] == 10
```

Reply:
let's do in a separate PR, I am checking why this test is failing. In my local environment it works.